ckanext/dalrrd_emc_dcpr/templates/snippets/facet_list.html

```diff
@@ -2,6 +2,8 @@
 
 {% block facet_list %}
     {% asset "ckanext-dalrrdemcdcpr/facets-active-js" %}
+    {% asset "ckanext-dalrrdemcdcpr/xml-parser-js" %}
+
```

Comment:
@Fanevanjanahary thanks for the file change, a quick question, the xml parser asset doesn't relate to facet_list, can we put this asset somewhere else more generic, like base.html or page.html or others ?